README.md

```diff
@@ -456,6 +456,10 @@ OPTIONS
   --access-token=access-token              Eclipse Che OIDC Access Token
 
   --listr-renderer=default|silent|verbose  [default: default] Listr renderer
+
+  --max-items=max-items                    [default: 30] The maximum number of workspaces to return
+
+  --skip-count=skip-count                  The number of workspaces to skip
```

Comment:
1. I don't time they're parameters for CLI... Could we check like how kubectl, oc does and do the same... Maybe it makes sense to fetch all workspaces page by page internally and show them?

2. If you think we need paging here, won't it be more convenient to use --page -p, --page-size?
like `workspace:list -p=3` instead `workspace:list --skip-count=90`, I know it's not an issue to calculate 30*3 in mind but it makes sense to let people learn math with other tools but not chectl.

I don't think users will make many workspaces, so my personal :+1: for item 1

Reply:
Make sense!